app/Http/Middleware/CheckPermissions.php

```diff
@@ -26,6 +26,6 @@ public function handle($request, Closure $next, $section = null)
 
         return response()->view('layouts/basic', [
             'content' => view('errors/403'),
-        ]);
+        ], 403);
```

Comment:
Subtle! Nice. Good catch.

Reply:
Thanks - it behaved properly to the user, but the tests weren't happy about it, and a 403 is the right response here. (Do not ask me how long it took me to figure that out.)